curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/use-capture-groups-to-search-and-replace.english.md

```diff
@@ -28,7 +28,7 @@ You can also access capture groups in the replacement string with dollar signs (
 
 ## Instructions
 <section id='instructions'>
-Write a regex so that it will search for the string <code>"good"</code>. Then update the <code>replaceText</code> variable to replace <code>"good"</code> with <code>"okey-dokey"</code>.
+Write a regex using three capture groups that will replace the string "one two three" with the string "three two one" and assign the result to the <code>replaceText</code> variable.
```

Comment:
> `the result to the <code>replaceText</code> variable.` 

Shouldn't this be the `result` variable?

Reply:
Good catch. Fix will be up momentarily.